app/pages/project/classify.jsx

```diff
@@ -249,7 +248,12 @@ export class ProjectClassifyPage extends React.Component {
         }
       }).then((subjects) => {
         const nonLoadedSubjects = subjects.filter(newSubject => newSubject !== subjectToLoad);
-        const filteredSubjects = nonLoadedSubjects.filter(nonLoadedSubject => !nonLoadedSubject.already_seen && !nonLoadedSubject.retired);
+        const filteredSubjects = nonLoadedSubjects.filter((nonLoadedSubject) => {
+          const notSeen = !nonLoadedSubject.already_seen &&
+            !nonLoadedSubject.retired &&
+            !seenThisSession(workflow, nonLoadedSubject);
```

Comment:
This is throwing a type error when I'm logged out. Looking over the function, I think this line needs to be `!seenThisSession.check(workflow, nonLoadedSubject)`

Reply:
Thanks for catching this. It should be fixed now.